src/Symfony/Bridge/Twig/Mime/Renderer.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Mime;
+
+use League\HTMLToMarkdown\HtmlConverter;
+use Twig\Environment;
+
+class Renderer
```

Comment:
No interface implementing. I mean one could create its own `FooRenderer` without extending this one? 🤔